src/components/10-atoms/input-text/index.react.js

```diff
@@ -16,6 +16,9 @@ export default createElement => ({
   required = false,
   disabled = false,
   checkMark = false,
+  maxLength = '',
```

Comment:
This must be `maxLength = 0`, since you are assigning a property here (not a string-valued attribute) and the declared property is of type `Number` (which is semantically correct).

Reply:
According to w3c, the default value is `524288`. I will stick with that, if that is ok for you.